.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: elixir
+elixir:
+  - 1.4
+addons:
+  postgresql: '9.4'
+env:
+  - MIX_ENV=test
+before_script:
+  - mix do ecto.create
+script:
+  - mix do deps.get, compile --warnings-as-errors, coveralls.json
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
+# See: github.com/dwyl/repo-badges#documentation
+after_script:
+  - MIX_ENV=docs mix deps.get
+  - MIX_ENV=docs mix inch.report
```

Comment:
`MIX_ENV=docs` is not a typical `MIX_ENV`. As far as I can see we are also doing nothing in that `MIX_ENV`, so it behaves just as `MIX_ENV=dev` would behave.
I was intregued as to the use case for `MIX_ENV=docs`, so doing a search for `MIX_ENV=docs` in github, I found that it is used in places like here: https://github.com/zamith/tomlex/blob/master/.travis.yml#L9
But that repo makes use of `MIX_ENV=docs` in the `mix.exs` file in that only the deps which are used for documentation are compiled. We are not doing that in our `mix.exs` file, so we should either remove `MIX_ENV=docs` or update our mix file to be similar to https://github.com/zamith/tomlex/blob/master/mix.exs